### PR TITLE
Webservice endpoint tags & other doc (Issue: MUR-9396)

### PR DESCRIPTION
**Pull request by sauvainr**

### CODE REVIEW

File: services/twilio.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+# Enable twilio service we no specific configuration: see http://docs.exosite.com/reference/services/twilio for more information
```
Comment (exjesper) [inline services/twilio.yaml]:
```suggestion
# Enable twilio service without a specific configuration: see http://docs.exosite.com/reference/services/twilio for more information
```

File: endpoints/example.lua [hunk 2]
```diff
@@ -1,5 +1,7 @@
---#ENDPOINT POST /api/user
-print("Creating a new user")
+--#ENDPOINT POST /api/user text/plain
+return "new-user-id"
 
 --#ENDPOINT GET /api/user/{userId}
+--#TAGS user
```
Comment (exjesper) [inline endpoints/example.lua]: Multiple tags by comma separated list?

Comment (sauvainr) [reply endpoints/example.lua]: yes or space.

Comment (sauvainr) [inline endpoints/example.lua]:
```suggestion
--#TAGS user public
```

File: services/webservice.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# webservice settings: see http://docs.exosite.com/reference/services/webservice for more information
+
+# Tag to filter which endpoints to display on the auto-generated documentation
+documented_endpoints:
+  - user
```
Comment (sauvainr) [inline services/webservice.yaml]:
```suggestion
  - public
```

File: endpoints/example.lua [hunk 4]
```diff
@@ -1,5 +1,7 @@
---#ENDPOINT POST /api/user
-print("Creating a new user")
+--#ENDPOINT POST /api/user text/plain
+return "new-user-id"
```
Comment (sauvainr) [inline endpoints/example.lua]:
```suggestion
-- a lua handler for this endpoint
return "new-user-id"
```

**Pull request merged**